### ISSUE TITLE
Database connection

### ISSUE THREAD

**Issue by TijoloAzul** [dinis/raspberryGameSuite#10]
Server should be able to communicate with a postgres database.

All migration files should be in a properties file.

**Comment by TijoloAzul** [comment]
Depends on #11

**Comment by TijoloAzul** [comment]
@diogodinisf 
if you prefer, I've two distinct suggestions:

1. in this task you build a nice structure that simplifies our life, but you do not need to have any dataaccess command in specific
2. you split this in two parts: i) you add dataAccess capacity and create three commands login, logout and register users ii) you build a nice structure and re-adapt the solution, keeping the interface

If you prefer 2, please create a new issue, and do a pull request as soon as possible.

**Comment by diogodinisf** [comment]
I have serious doubts in the implementation of permanent data access. I will need some time to struct my ideias for this. 
By the way, at the moment I have some atomic data access, like register or name verify. I will do something today's afternoon.

**Comment by TijoloAzul** [comment]
So perhaps it's best to split your task in two parts... 
Meanwhile I can work over half of your work and ask some colleagues about the subject

**Comment by diogodinisf** [comment]
I have some work on database's branch, you can see and comment

**Comment by diogodinisf** [comment]
See the last commit. Give-me your feedback and if this is the right way we can work on it.

**Comment by TijoloAzul** [comment]
It seems a good candidate.
Tomorrow I'll talk to some friends about it, and investigate a little more.

I was reading stuff about transaction management, e.g.:
http://simplejta.sourceforge.net/

**Comment by TijoloAzul** [comment]
Em relação a connections e transações
http://stackoverflow.com/questions/4940648/how-to-start-a-transaction-in-jdbc

A conecção abre em modo auto-commit (cada acesso é uma transação)
Logo tu queres: abrir a conecção e desligar o autocommit

**Comment by diogodinisf** [comment]
I see, thank you.

**Comment by TijoloAzul** [comment]
I've officially stlitted this task, see #15
